run_cp.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+cp dev.env.default .env
+temp="`docker inspect -f '{{range .NetworkSettings.Networks}}{{.Gateway}}{{end}}' tapp_rails-app_1`"
```

Comment:
yikes! this is heroic, but probably only will serve to make prototype the endpoint access from cp to tapp. (It makes me  wonder if keeping the images separate is worth the kind of hypothetical security benefit)